content/python/concepts/built-in-functions/terms/tuple/tuple.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'tuple()'
+Description: 'Creates a new tuple.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Collections'
+  - 'Tuples'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The built-in **`tuple()`** function creates a new [tuple](https://www.codecademy.com/resources/docs/python/tuples).
+
+## Syntax
+
+```pseudo
+tuple(iterable)
+```
+
+The `iterable` parameter is optional and is an object that can be looped over, such as a [list](https://www.codecademy.com/resources/docs/python/lists) or [string](https://www.codecademy.com/resources/docs/python/strings).
+
+## Codebyte Example
+
+The following example showcases how the `tuple()` function is used to create a new tuple:
+
+```codebyte/python
+my_tuple = tuple((1, 2, 3))
```

Comment:
Are the inside parenthesis optional? If yes, can we include something about it

Reply:
The inside parentheses are not optional. This line indicates that you are trying to create a tuple out of another tuple. I think it would be better to use the tuple function to create a tuple out of a list or other collection data struct.

```py
my_list = [1, 2, 3]
my_tuple = tuple(my_list)
```